.github/workflows/release.yml

```diff
@@ -0,0 +1,46 @@
+name: Release Model Card Toolkit package to PyPI and TestPyPI
+
+on:
+  release:
+    types: [published]
+    tags:
+      - v*
+
+jobs:
+ build-and-publish:
+    name: Build Model Card Toolkit package and release to PyPI and Test PyPI
+    runs-on: ubuntu-latest
+    container:
+      image: gcr.io/tfx-oss-public/tfx_base:py37-20200729
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Build wheel
+      run: |
+        bash -x package_build/initialize.sh; \
+        CFLAGS=$(/usr/bin/python-config --cflags) \
+          python package_build/model-card-toolkit/setup.py bdist_wheel; \
+        MCT_WHEEL=$(find dist -name "model_card_toolkit-*.whl"); \
+
+    - name: Publish Model Card Toolkit distribution package to Test PyPI
+      uses: pypa/gh-action-pypi-publish@v1.5.0
+      with:
+        password: ${{ secrets.TEST_PYPI_API_TOKEN }}
```

Comment:
I'll ping Ahmed again, in case I haven't heard from him by Monday.

Reply:
Thank you!